### DISCUSSION TITLE
Matter (smart home standard) plugin

### DISCUSSION THREAD

**Discussion by alteredtech** [grafana/grafana#58688] [Plugins]
With people using grafana to monitor their home with grafana. A plug-in to bridge between grafana and matter would be incredible for the grafana ecosystem with its already incredible list of plugins.

The usage of seeing data points from our dashboard in our personal smart home to add additional control of things.
Thermostats from custom sensors. Start ac if servers start getting too hot. Power usage of the house and if it reaches a point, turn off certain devices. So much more that I can't even fathom.

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
